### ISSUE TITLE
Feature Request: prevent modifications to the "created" index within the Pocketbase UI

### ISSUE THREAD

**Issue by jonshipman** [pocketbase/pocketbase#3871]
Currently it's possible to remove or modify the "created" index in collections. I was wondering if this shouldn't be warned against or disabled from the UI altogether. Disabling would be easier similar to how id, modified, and created columns are required table columns.

If the "created" index is modified (either joined with another column or removed) the admin listing can fail to load under the default timeout of 30s with large collections. Ran into this issue myself with a table that was over 140,000 records and 7GB. Merged the created and active columns for reasons, and eventually the admin UI failed to load. Working with @ganigeorgiev we discovered that it was the combination of a large table and missing created index.

Since created is the default sort in the admin and is a default column, it stands to reason the index "created" should also be treated similarly.

**Comment by ganigeorgiev** [comment]
No, I don't think this is a good idea. Users can sort by any root level collection field, not just `created` and its up to the developer to create and manage the indexes they think they need. If this action result to a slow or inefficient query, users will have to adjust them like in any other database.

Also note that the issue in your case is mostly related not to the number of db entries (140k is not that much) but more because of the large size of the individual db record.
